.github/workflows/main.yml

```diff
@@ -0,0 +1,78 @@
+name: CI
+on:
+- pull_request
+- push
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+        - python-version: 3
+          env:
+            TOXENV: docs
+
```

Comment:
**nitpick**: Do we need a line break here? :-) We have others in the next lines too.